br/pkg/lightning/backend/external/engine.go

```diff
@@ -257,6 +374,7 @@ func (m *MemoryIngestData) GetTS() uint64 {
 
 // Finish implements IngestData.Finish.
 func (m *MemoryIngestData) Finish(totalBytes, totalCount int64) {
-	//TODO implement me
-	panic("implement me")
+	m.importedKVSize.Add(totalBytes)
+	m.importedKVCount.Add(totalCount)
+	m.memBuf.Destroy()
```

Comment:
Is there a situation where "MemoryIngestData.Finish()" is called concurrently?

Reply:
Currently one regionJob should be called its `ingestData.Finish` once